client/maps/actions.ts

```diff
@@ -90,3 +91,12 @@ export interface LoadMapInfo {
   type: '@maps/loadMapInfo'
   payload: MapInfoJson
 }
+
+/**
+ * Loads multiple map infos directly. Useful if map infos are received from a source that doesn't
+ * dispatch through redux.
+ */
+export interface LoadMapInfos {
```

Comment:
dunno how many places actually use `LoadMapInfo` but it doesn't really seem like we need both of these, those places could just use this